src/py/flwr/server/compat/app.py

```diff
@@ -79,10 +79,14 @@ def start_driver(  # pylint: disable=too-many-arguments, too-many-locals
     log(INFO, "")
 
     # Start the thread updating nodes
-    thread, f_stop = start_update_client_manager_thread(
+    thread, f_stop, c_done = start_update_client_manager_thread(
         driver, initialized_server.client_manager()
     )
 
+    # Wait until the node registration done
+    if not c_done.is_set():
+        c_done.wait()
```

Comment:
```suggestion
    c_done.wait()
```

Reply:
why not leaving it like this?

Reply:
then, let' apply the same style to `app_utils_test.py` and `default_workflows.py`